tests/integration/install/test_installation.py

```diff
@@ -356,74 +354,6 @@ def test_check_inventory_database_exists(ws, installation_ctx):
         installation_ctx.workspace_installer.configure()
 
 
-@retried(on=[NotFound], timeout=timedelta(minutes=5))
-@pytest.mark.parametrize('prepare_tables_for_migration', [('regular')], indirect=True)
-def test_table_migration_job(ws, installation_ctx, env_or_skip, prepare_tables_for_migration):
```

Comment:
Where do these tests go?

Reply:
they are failing too much in our test runs and they don't cover any critical scenario that is already covered by other tests.